include/rangers-api/Application/Text/AppTextListener.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+namespace app::text {
+    class AppTextListener : public hh::ui::TextListenerObject {
```

Comment:
Starting braces should be started on a line to keep consistent with the original repository.